core/chains/evm/config/chain_specific_config.go

```diff
@@ -325,6 +325,20 @@ func setChainSpecificConfigDefaultSets() {
 	optimismGoerli := optimismKovan
 	optimismGoerli.linkContractAddress = "0xdc2CC710e42857672E7907CF474a69B63B93089f"
 
+	// Optimism's Bedrock upgrade uses EIP-1559.
+	optimismBedrock := fallbackDefaultSet
+	optimismBedrock.eip1559DynamicFees = true
+	optimismBedrock.blockEmissionIdleWarningThreshold = 30 * time.Second
+	optimismBedrock.nodeDeadAfterNoNewHeadersThreshold = 60 * time.Second
+	optimismBedrock.blockHistoryEstimatorBlockHistorySize = 24
+	optimismBedrock.chainType = config.ChainOptimismBedrock
+	optimismBedrock.ethTxResendAfterThreshold = 30 * time.Second
+	optimismBedrock.finalityDepth = 200				 // Although Bedrock is re-org resistant for 10 blocks on L1, L2 blocks are considered final when they are included in L2 final block (PoS).
+	optimismBedrock.headTrackerHistoryDepth = 300
+	optimismBedrock.linkContractAddress = "0x0000000000000000000000000000000000000000"
```

Comment:
Leave it as an empty string unless it's actually a valid contract address

Reply:
Empty string won't work because v2 expects a valid EIP155 Address. I can remove it completely though.